.github/workflows/ci.yml

```diff
@@ -13,8 +13,6 @@ env:
   # 7 GiB by default on GitHub, setting to 6 GiB
   # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
   NODE_OPTIONS: --max-old-space-size=6144
-  # install playwright binary manually (because pnpm only runs install script once)
-  PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: "1"
```

Comment:
drive-by: Playwright does not install browsers as part of `install script` [since v1.38](https://playwright.dev/docs/release-notes#version-138).